### PR TITLE
Implement cheatcodes compatible with Medusa in fuzzlib

### PR DESCRIPTION
**Pull request by vishnuram1999**
Currently, `fuzzlib` supports cheatcodes specifically designed for the `Echidna` fuzzer. This pull request proposes introducing a new set of cheatcodes compatible with the `Medusa` fuzzer.

Based on the Medusa documentation on [cheatcodes](https://github.com/crytic/medusa/tree/dev/mdbook/docs/src/cheatcodes), `IStdCheats` Interface contains the list of functions that can be [used](https://github.com/crytic/medusa/blob/dev/mdbook/docs/src/cheatcodes_overview.md). 

This results in **improved compatibility**. Developers/Auditors can utilize a wider range of cheatcodes, regardless of the chosen fuzzer (`Echidna` or `Medusa`).

This Pull Request addresses #9 issue.

### CODE REVIEW

**Comment by rappie** [comment]
Thank you for the PR! I will review it soon :slightly_smiling_face:

File: src/FuzzBase.sol [hunk 1]
```diff
@@ -6,7 +6,9 @@ import "./Constants.sol";
 import "./AssertWrapper.sol";
 import "./ClampWrapper.sol";
 import "./IHevm.sol";
+import "./IStdCheats.sol";
 
 abstract contract FuzzBase is AssertWrapper, ClampWrapper, Logging, Constants {
-    IHevm internal vm = IHevm(0x7109709ECfa91a80626fF3989D68f67F5b1DD12D);
+    IHevm internal vm = IHevm(0x7109709ECfa91a80626fF3989D68f67F5b1DD12D); // for echidna
+    IStdCheats internal cheats = IStdCheats(0x7109709ECfa91a80626fF3989D68f67F5b1DD12D); // for medusa
```
Comment (rappie) [inline src/FuzzBase.sol:13]: i would like to have some unified interface to `vm` in the longterm, but for now it makes sense to have two

what about renaming `cheats` to `mvm`? (medusa vm)

we could also rename `vm` but i prefer to keep it the same for compatibility reasons

**Review by rappie — CHANGES_REQUESTED** [review]

File: src/FuzzBase.sol (hunk 1 above)
Comment (ljz3) [reply src/FuzzBase.sol:13]: I agree with this🙂

Comment (vishnuram1999) [reply src/FuzzBase.sol:13]: I agree too, I will rename `cheats` to `mvm`.

**Review by rappie — APPROVED** [review]

**Review by ljz3 — APPROVED** [review]

**Pull request merged**